src/test/testDiagnotics.test.ts

```diff
@@ -0,0 +1,96 @@
+import * as vscode from 'vscode';
+import {
+  getDocUri,
+  activate,
+  testDiagnostics,
+  sleep,
+  updateSettings,
+} from './helper';
+
+describe('TEST FOR DIAGNOSTICS IN LOCAL ENVIRONMENT (without ee)', () => {
+  const docUri1 = getDocUri('diagnostics/1.yml');
+  const docUri2 = getDocUri('diagnostics/2.yml');
+
+  beforeEach(async () => {
+    await vscode.commands.executeCommand('workbench.action.closeAllEditors');
+  });
+
+  describe('Diagnostic test with ansible-lint', () => {
+    it('Test 1', async () => {
```

Comment:
The test names should contain a meaningful continuation of `it` and probably use the word `should`. So that it'd read `it should do/result in something`.
```suggestion
    it('should result in a result', async () => {
```

Reply:
I do agree that "test 1" does not say anything meaningful but neither `should result in a result` ;)

Reply:
I agree. I did not intend to have this message. It was only for testing purpose and then I forgot to remove from before pushing the PR.

Reply:
Done !

Reply:
> I do agree that "test 1" does not say anything meaningful but neither `should result in a result` ;)

Totally, it was mostly to demonstrate how hard it is to guess what the test expectation is if the author does not provide it.